src/openrct2/Game.h

```diff
@@ -89,7 +89,8 @@ enum GAME_COMMAND
     GAME_COMMAND_MODIFY_TILE,
     GAME_COMMAND_EDIT_SCENARIO_OPTIONS,
     GAME_COMMAND_PLACE_PEEP_SPAWN, // GA, TODO: refactor to separate array for just game actions
-    GAME_COMMAND_COUNT
+    GAME_COMMAND_SET_CLIMATE,
```

Comment:
TODO: add `GA` comment (waiting for CI to turn (hopefully) turn green first.
```suggestion
    GAME_COMMAND_SET_CLIMATE, // GA
```